vars/skipStage.groovy

```diff
@@ -545,22 +557,22 @@ boolean call(Map config = [:]) {
                    skip_if_unstable()
         case 'Functional_Hardware_Small':
         case 'Functional Hardware Small':
-            return skip_ftest_hw('small', target_branch)
+            return skip_ftest_hw('small', 'el8', target_branch)
```

Comment:
Why are we putting a distro here?
While we are currently using el-8, this looks like something that will need to be addressed when we move to el-9.

Reply:
I've added skipping the HW stages if we skipped building  the RPMs that they require, similar to what we curently do in `skip_ftest()` for the functional VM stages.  While thios isn't much different than what we do when we call `skip_ftest_valgrind('el8', target_branch)` for example, I could see an improvement to use a variable.  The `'target'` from parseStageInfo be the correct value to use here, right?

Reply:
Yes.  Ideally we would not need a parameter as we should know that the specific rpm build was skipped.

Reply:
Ultimately I think you need to utilize [hwDistroTarget()](https://github.com/daos-stack/pipeline-lib/blob/668ba308eadd4cc9980cf758190c752e1e073546/vars/hwDistroTarget.groovy) here.

Reply:
I've updated this to use `hwDistroTarget2()`